plugins/notification-mappers/src/index.ts

```diff
@@ -98,35 +98,116 @@ function createSpaceInfo<T extends Space> (
   return data
 }
 
+function addSpaceMember (
+  ctx: TxUpdateDoc<Space> | TxCreateDoc<Space>,
+  options: MappingOptions,
+  member: Ref<Account>,
+  result: SpaceNotifications[]
+): void {
+  const spaceNotification = createSpaceNotification(ctx, options, member)
+  if (result.findIndex((p) => p._id === spaceNotification._id) === -1) {
+    result.push(spaceNotification)
+  }
+}
+
+async function createDocSpaceInfoHandler (ctx: TxCreateDoc<Doc>, options: MappingOptions): Promise<SpaceInfo[]> {
+  if (options.hierarchy.isDerived(ctx.objectClass, core.class.Space)) {
+    return [createSpaceInfo(ctx as TxCreateDoc<Space>, options.descriptor._id)]
+  }
+  await tryUpdateSpaceInfo(ctx, options)
+  return []
```

Comment:
Why we not return updated value, does DD processor will diff and remove existing DD? Should we return updated value as well? I've created ticked #395 to disable processor for doing anything if mapper is designed for it,

Reply:
Because SpaceInfo is derived data from space, not from object in.